daemons/based/pacemaker-based.h

```diff
@@ -123,6 +125,14 @@ int cib_process_upgrade_server(const char *op, int options, const char *section,
                                xmlNode *req, xmlNode *input,
                                xmlNode *existing_cib, xmlNode **result_cib,
                                xmlNode **answer);
+int cib_process_init_transaction(const char *op, int options,
```

Comment:
These could reasonably go in `based_transaction.h` but I kept them with the other `cib_process_*` functions.